tests/lit-tests/3066.ispc

```diff
@@ -0,0 +1,8 @@
+// RUN: %{ispc} --target=host %s -M -MT target_name 2>&1 | FileCheck %s
```

Comment:
Did you meant to add `--target-os=windows`?

Alternatively you can have `WINDOWS_HOST` instead of `WINDOWS_ENABLED`

Reply:
```suggestion
// RUN: %{ispc} --target=host --target-os=windows %s -M -MT target_name 2>&1 | FileCheck %s
```